inc/sai.h

```diff
@@ -31,7 +31,7 @@
 #include "saibuffer.h"
 #include "saifdb.h"
 #include "saihash.h"
-#include "saihostintf.h"
+#include "saihostif.h"
```

Comment:
thats how api_t are called, this change will have no impact on current vendor source code, but if we change name more complicated changes maybe required on vendor side. @lguohan can you check this change and suggest ?